getting-started/vs-integration/new-project-wizard.md

```diff
@@ -50,31 +52,28 @@ To start the wizard, use either of the following options:
 
 ## Using the Wizard
 
-In the order of their appearance, the Create New Project wizard presents the following screens with options that will assist you to configure your new app:
-
-1. On the first screen, you can select a location and a name for your project.
+In the order of their appearance, the Create New Project wizard presents several screens with options that will assist you to configure your new app.
 
-2. On the second screen, you can choose:
+After you select the **Create New Project** option from the extensions menu, Visual Studio will open a **Configure your new project** window where you can select a location and a name for your project. 
 
-   * A WebAssembly, server-side, or Hybrid Blazor app.
-   * The .NET version you want to use.
-   * The Telerik UI for Blazor version.
-   * One of the [predefined Telerik project](#predefined-project-templates) templates.
+Clicking the **Create** button in this window will open the wizard and you will go through the following steps:
 
-   ![The Create New Project Wizard Templates Options](images/vsx-extension-base.png)
+### Step 1: Select Initial Project Configuration
 
-3. On the third screen, you can choose from different [themes]({%slug general-information/themes%}).
+In this screen, you can choose:
```

Comment:
* Make sure that all bullet items use consistent ending punctuation.
* Switch hyphens to `&mdash;`, no spaces around it.